django/contrib/messages/test.py

```diff
@@ -0,0 +1,26 @@
+from .api import get_messages
+from .utils import get_level_tags
+
+
+class MessagesTestMixin:
+    def assertMessages(self, response, expected_messages):
+        level_to_name = get_level_tags()
+
+        def level_name(level):
+            level_name = level_to_name.get(level, level)
+            if isinstance(level_name, str) and level_name:
+                return level_name.upper()
+            return level
```

Comment:
Checking `level_name` seems to be unnecessary:
```suggestion
            return level_name.upper() if isinstance(level_name, str) else level_name
```

Reply:
That’s correct. I had the case `MESSAGE_TAGS={constants.INFO: ""}` in mind, but it works just as well with an empty string.